src/models.py

```diff
@@ -23,8 +23,21 @@ def __init__(self, file_path: str, exit_diff: Diff, diffs: List[Diff]):
         self.exit_diff = exit_diff
         self.diffs = diffs
 class Metadata:
```

Comment:
Not sure about adding external data to a model that's supposed to match the JSON structure

Reply:
I [split the `Metadata` model in `TestOutputMetadata` and `PageGenSettings`](https://github.com/chocoteam/PerformanceAction/pull/34/commits/16fb3616da449ed97fbb87231cdd59497aa55952).